apps/meteor/app/lib/server/lib/notifyListener.ts

```diff
@@ -87,6 +89,19 @@ export const notifyOnRoomChangedByUsernamesOrUids = withDbWatcherCheck(
 	},
 );
 
+export const notifyOnRoomChangedByContactId = withDbWatcherCheck(
+	async <T extends IOmnichannelRoom>(
+		contactId: Exclude<T['contactId'], undefined>,
```

Comment:
Do you actually need the generic here? I see this more as an `string`, or even fancier, `ILivechatContact['_id']`

wdyt?